install/index.php

```diff
@@ -1847,7 +1845,7 @@ function revertSetting(defval, inpid)
 
 		if(substr($cookiedomain, 0, 4) == "www.")
 		{
-			$cookiedomain = my_substr($cookiedomain, 4);
+			$cookiedomain = substr($cookiedomain, 4);
```

Comment:
I don't think that's how it works. When you specify that you want the first 4 characters, substr doesn't take into account that the length of multibyte characters is not one byte therefore the result may not be right. I certainly don't know how the inner workings of substr so I can't be sure about that. I don't think it hurts to use my_substr here though - unless of course you know for sure this is fine.